src/main/java/calculator/SimpleCalculator.java

```diff
@@ -0,0 +1,110 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class SimpleCalculator {
+	private static final List<Character> opList = new ArrayList<>(Arrays.asList('+', '-', '*', '/'));
+	List<String> splitExpressionList = new ArrayList<>();
+
+	public Integer selectOperand(String s, int idx) {
+		return Integer.valueOf(s.split(" ")[idx]);
+	}
+
+	public String plus(String s) {
+		return (selectOperand(s, 0) + selectOperand(s, 2)) + "";
+	}
+
+	public String minus(String s) {
+		return (selectOperand(s, 0) - selectOperand(s, 2)) + "";
+	}
+
+	public String multiply(String s) {
+		return (selectOperand(s, 0) * selectOperand(s, 2)) + "";
+	}
+
+	public String divide(String s) throws NotEqualRemainderZero {
+		int firstOperand  = selectOperand(s, 0);
+		int secondOperand = selectOperand(s, 2);
+		if (firstOperand % secondOperand != 0) {
+			throw new NotEqualRemainderZero();
+		}
+		return (selectOperand(s, 0) / selectOperand(s, 2)) + "";
```

Comment:
같은연산을 이미 수행해줬는데, 똑같은 연산이 수행되고 있네요.
```suggestion
		return firstOperand / secondOperand + "";
```

Reply:
ㅜ.ㅜ 네, 맞습니다.
제안해주신 코드 감사합니다.